config/default_config.json

```diff
@@ -17,7 +17,8 @@
     "Public-Domain",
     "Unlicense",
     "WTFPL",
-    "X11"
+    "X11",
+    "zlib"
```

Comment:
The SPDX identifier for this license is capitalized (`Zlib`):
https://spdx.org/licenses/Zlib.html

Reply:
Whoops good catch